torch/_dynamo/trace_rules.py

```diff
@@ -215,8 +216,11 @@
     "torch.autograd.forward_ad.unpack_dual": UserFunctionVariable,
     # functorch/jacfwd
     "torch._functorch.eager_transforms.jacfwd": UserFunctionVariable,
+    "torch._functorch.eager_transforms.jacfwd_impl": UserFunctionVariable,
```

Comment:
There's no more jacrev_impl and jacfwd_impl right?

Reply:
No! I'll remove this.